skiko/buildSrc/src/main/kotlin/tasks/configuration/CommonTasksConfiguration.kt

```diff
@@ -134,4 +142,46 @@ fun Project.configureSignAndPublishDependencies() {
             }
         }
     }
+}
+
+fun KotlinTarget.generateVersion(
```

Comment:
Why this function is moved?

Reply:
it's used in all targets but by some reason during the build.gradle.kts split it migrated to NativeTasks